frontend/packages/client/src/components/UserProfile/ShareProfileDropdown.tsx

```diff
@@ -0,0 +1,27 @@
+import { ShareDropdown } from 'components';
+import { FRONTEND_URL } from 'const';
+import { Box } from '@chakra-ui/react';
+
+interface ShareProfileDropdownProps {
+  userAddr: string;
+}
+const ShareProfileDropdown: React.FC<ShareProfileDropdownProps> = ({
+  userAddr,
+}) => {
+  const profileUrl = `${FRONTEND_URL}/#/profile?addr=${userAddr}`;
+
+  const twitterPost = `https://twitter.com/intent/tweet?text=${encodeURIComponent(
+    `Check at my profile in ${profileUrl} on CAST! `
```

Comment:
Same thing here as well I don't think you need the url in the string because `ShareDropdown` uses the twitter url and encodes the string passed to the component